src/commands/settings/AboutCommand.ts

```diff
@@ -38,33 +38,31 @@ export default class AboutCommand extends BaseCommand {
                 new MessageEmbed()
                 .setAuthor({ iconURL: client.user?.displayAvatarURL(), name: "SudoBot" })
                 .setDescription(`
-                    A free and open source Discord moderation bot, specially created for **The Everything Server**.
-
-                    Copyright (C) ${new Date().getFullYear()} OSN Inc.
-                    This bot comes with ABSOLUTELY NO WARRANTY.
-                    This is free software, and you are welcome to redistribute it under certain conditions.
-                    See the [GNU Affero General Public License v3](https://www.gnu.org/licenses/agpl-3.0.en.html) for more detailed information.
+                    __**A free and open source Discord moderation bot, specially created for The Everything Server**__.
```

Comment:
Maybe add a newline after this line?

Reply:
It looks fine to me with the ">".

Reply:
Alright

Reply:
> It looks fine to me with the ">".

But remove those from the fields please.